### PR TITLE
UCT/IB: Detect 50g Eth speed.

### LINKED ISSUE #1813: uct_ib_iface_query() does not recognize 50 Gbit eth speed
ib_iface.c:749  UCX  ERROR Invalid active_speed on mlx5_0:1: 64
also the function should either return void or callers must check its return value...

### PR DESCRIPTION
**Pull request by yosefe**
Fixes #1813

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4668/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4672/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@alinask pls review

File: src/uct/ib/base/ib_iface.c [hunk 1]
```diff
@@ -740,11 +740,16 @@ ucs_status_t uct_ib_iface_query(uct_ib_iface_t *iface, size_t xport_hdr_len,
...
         encoding                     = 64.0/66.0;
         break;
-    case 32: /* EDR */
+    case 32: /* EDR / 100g Eth */
         iface_attr->latency.overhead = 600e-9;
         signal_rate                  = 25.78125e9;
         encoding                     = 64.0/66.0;
         break;
+    case 64: /* 50g Eth */
+        iface_attr->latency.overhead = 600e-9;
```
Comment (alinask) [inline src/uct/ib/base/ib_iface.c]: same overhead as EDR?

**Comment by alinask** [comment]
@yosefe Shouldn't this function's return status not be ignored?

File: src/uct/ib/base/ib_iface.c (hunk 1 above)
Comment (yosefe) [reply src/uct/ib/base/ib_iface.c]: yes, should be close enough

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4703/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@alinask fixed

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4704/ for details (Mellanox internal link).

**Pull request merged**